matlab/reco.m

```diff
@@ -20,7 +20,7 @@
 S = h5read(filenameSM, '/measurement/data');
 
 % reinterpret as complex numbers
-S = squeeze(S(1,:,:,:,:) + 1i*S(2,:,:,:,:));
+S = complex(S.r,S.i);
```

Comment:
@NeumannIMT: Is it that simple? Are you using the new data that I uploaded last night?